keras_cv/layers/preprocessing/rgb_shift.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+class RGBShift(layers.Layer):
+    """RGBShift class randomly shift values for each channel of the 
+    input RGB image. The expected images should be `(0-255)` pixel ranges.
+
+    Input shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `channels_last` format
+    Output shape:
+        3D (unbatched) or 4D (batched) tensor with shape:
+        `(..., height, width, channels)`, in `channels_last` format
+
+    Args:
+        factor: A scalar or tuple or list of two upper and lower bound 
+            number. If factor is a single value, the range will be 
+            `(-factor, factor)`. The factor value can be float or integer; 
+            for float the valid limits are `(-1.0, 1.0)` and for integer the 
+            valid limits are `(-255, 255)`.
+        seed: Integer. Used to create a random seed. Default: None.
+
+    Call arguments: 
+        images: Tensor representing images of shape
+            `(..., height, width, channels)`, with dtype tf.float32 / tf.uint8, or,
+            `(height, width, channels)`, with dtype tf.float32 / tf.uint8
+        training: A boolean argument that determines whether the call should be run 
+            in inference mode or training mode. Default: True.
+   
+    Usage:
+    ```python
+    (images, labels), _ = tf.keras.datasets.cifar10.load_data()
+    rgb_shift = keras_cv.layers.RGBShift(factor=(-2, 2))
+    augmented_images = rgb_shift(images)
+    ```
+    """
+
+    _FACTOR_VALIDATION_ERROR = (
+        "The factor should be a scalar, "
+        "a tuple or a list of two upper and lower "
+        "bound values in the range `(-1.0, 1.0)` as float or `(-255, 255) as integer."
+    )
+
+    def __init__(self, factor, seed=None, **kwargs):
+        super().__init__(**kwargs)
+        self.factor = self._set_factor_limit(factor)
+        self.seed = seed
+
+    def _set_factor_limit(self, factor):
+        if isinstance(factor, (tuple, list)):
+            if len(factor) != 2:
+                raise ValueError(
+                    self._FACTOR_VALIDATION_ERROR + f"Received: factor={factor}"
```

Comment:
You're missing a space after "integer." and before "Received:"

This applies to all locations of this error message.

Reply:
Didn't get the integer part. Is it:

```
if len(factor) != 2 : spacing after int ??
```

Reply:
Replace
```
self._FACTOR_VALIDATION_ERROR + f"Received: factor={factor}"
```
with
```
self._FACTOR_VALIDATION_ERROR + f" Received: factor={factor}"
```

At every location this line appears.

Reply:
Uh, I've already did that. I was confused a bit when you said ' after  "integer" ' .  Placing a space before 'Received' was done in line 69, 77, 84, 90. And in line 96, a space was set in previous line. So, I think it's ok.